### PR TITLE
Store EnergyCal in Spectrum

### PR DESCRIPTION
**Pull request by jvavrek**
Fix the auto merge of #229 

> I'm opening this since I think we talked about getting #180 soon as well. Is there anything other than simply adding a self.cal to Spectrum? Perhaps the rebin or combine_bins change the calibration, depending on how the calibration is defined?
> 
> Thoughts @jccurtis @markbandstra @cosama ?

### CODE REVIEW

File: becquerel/core/spectrum.py [hunk 1]
```diff
@@ -140,6 +140,7 @@ def __init__(self, counts=None, cps=None, uncs=None, bin_edges_kev=None,
         self._cps = None
         self._bin_edges_kev = None
         self._bin_edges_raw = None
+        self.cal = None
```
Comment (markbandstra) [inline becquerel/core/spectrum.py:143]: Rename to `energy_cal` since we will add a resolution calibration later.

**Comment by jvavrek** [comment]
Probably should add a test or two

**Comment by jvavrek** [comment]
@markbandstra I think this should be good to go once the pipeline passes

**Review by markbandstra — APPROVED** [review]
Looks good!

**Pull request merged**